File: Labs/Lab 1/lab1.py
nums=[0,1,2,3,7,8,9]
target=12
pairs={}
for i in nums:
    pairs[i]=target-i
first=0
found=False
for j in nums:
    for k in range(len(nums)):
        if(pairs[j]==nums[k]):
            print(first,k)
            found=True
            break
    if found:
        break
    first+=1

File: Lab1.java
package lab_1;

import java.util.HashMap;

public class Lab1 {
	public static void main(String args[]) {
	int[] nums= {0,1,2,3,7,8,9};
	int target=12;
	HashMap<Integer, Integer> pairs=new HashMap<Integer, Integer>();
	
	for(int i=0; i<nums.length; i++) {
		pairs.put(nums[i], target-nums[i]);
	}
	
	int first=0;
	for(int j:nums) {
		for(int k=1;k<nums.length;k++) {
		if(pairs.get(j)==nums[k]) {
			System.out.println(first+" "+k);
			return;
		}
	}
		first++;
	}
	}
}
